README.md

```diff
@@ -7,8 +7,9 @@ With this template you can collect different disk statistics.
 
 Installation
 ------------
-To install, copy `userparameter_diskstats.conf` to /etc/zabbix/zabbix_agentd.d/userparameter_diskstats.conf and `lld-disks.py` to /var/lib/zabbix/scripts/lld-disks.py.
-If /var/lib/zabbix/scripts does't exists, create it: ```sudo mkdir /var/lib/zabbix/scripts -p && sudo chown zabbix:zabbix /var/lib/zabbix -R```
+To install, copy `userparameter_diskstats.conf` to /etc/zabbix/zabbix_agentd.d/userparameter_diskstats.conf and `lld-disks.py` to `/usr/local/bin/lld-disks.py`.
```

Comment:
May we save original place of the script, as other users could already have it in it?